.dockerignore

```diff
@@ -1,4 +1,5 @@
 target/
```

Comment:
Trash this fine now that we only copy one binary

Reply:
Without this, copying over the context takes long because it has to copy the whole target folder which is usually 3GB or something.

Reply:
why does it copy over the context??

Reply:
@thomaseizinger please clarify this point.